sunpy/lightcurve/sources/het.py

```diff
@@ -0,0 +1,96 @@
+                'Column 12: Proton flux, 17.0-19.3 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 13: Uncertainty (sigma) for 17.0-19.3 MeV proton flux',
+                'Column 14: Proton flux, 20.8-23.8 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 15: Uncertainty (sigma) for 20.8-23.8 MeV proton flux',
+                'Column 16: Proton flux, 23.8-26.4 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 17: Uncertainty (sigma) for 23.8-26.4 MeV proton flux',
+                'Column 18: Proton flux, 26.3-29.7 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 19: Uncertainty (sigma) for 26.3-29.7 MeV proton flux',
+                'Column 20: Proton flux, 29.5-33.4 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 21: Uncertainty (sigma) for 29.5-33.4 MeV proton flux' ,
+                'Column 22: Proton flux, 33.4-35.8 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 23: Uncertainty (sigma) for 33.4-35.8 MeV proton flux' ,
+                'Column 24: Proton flux, 35.5-40.5 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 25: Uncertainty (sigma) for 35.5-40.5 MeV proton flux' ,
+                'Column 26: Proton flux, 40.0-60.0 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 27: Uncertainty (sigma) for 40.0-60.0 MeV proton flux' ,
+                'Column 28: Proton flux, 60.0-100.0 MeV, particles/(cm2-sr-sec-MeV)',
+                'Column 29: Uncertainty (sigma) for 60.0-100.0 MeV proton flux' ]
+   
+
+    data_modify = []
+    month_dict = {'Jan':1,'Feb':2,'Mar':3,'Apr':4,'May':5,'Jun':6,'Jul':7,'Aug':8,'Sep':9,'Oct':10,'Nov':11,'Dec':12}
+
+
+    #Adding Time Column based on type of data
+    if type_of_data == '1min':
+        header = ['Verse Number', 'DateTime'] + header
+        
+        #Converting start datetime separate columns into a single datetime column
+        for i in range(len(data)): 
+            date = datetime(data['col2'][i], month_dict[ data['col3'][i] ], data['col4'][i], int(("%04d" % (data['col5'][i],))[:2]), int(("%04d" % (data['col5'][i],))[2:]) )
+            data_modify = data_modify + [date]
+
+        data.remove_columns(['col2','col3','col4','col5'])
+    else:
+        header = ['Verse Number','TimeRange'] + header
+
+        #Converting start datetime and end datetime separate columns into a single TimeRange column
+        for i in range(len(data)): 
+            date1 = datetime(data['col2'][i], month_dict[ data['col3'][i] ], data['col4'][i], int(("%04d" % (data['col5'][i],))[:2]), int(("%04d" % (data['col5'][i],))[2:]) )
+            date2 = datetime(data['col6'][i], month_dict[ data['col7'][i] ], data['col8'][i], int(("%04d" % (data['col9'][i],))[:2]), int(("%04d" % (data['col9'][i],))[2:]) )
+            
+            #I don't know why but including it outside interferes with the upper case and results in segmetation fault 11 :/
+            from sunpy.time import TimeRange
```

Comment:
We try to conform to the [PEP 8 style guide for imports](https://www.python.org/dev/peps/pep-0008/#imports), and it seems especially egregious to bury an import this deep in code.  Style guideline aside, you or someone else needs to ferret out the bug that leads to the segmentation fault rather than sidestepping it, particularly because it's not apparent how this solution even works.